microshift_networking/microshift-configuring-routes.adoc

```diff
@@ -0,0 +1,52 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-configuring-routes"]
+= Configuring routes
+include::_attributes/attributes-microshift.adoc[]
+include::_attributes/common-attributes.adoc[]
+:context: microshift-configuring-routes
+
+toc::[]
+
+You can configure routes in {microshift-short} for networked clusters.
+//Q: assume this is not applicable to anything that is disconnected or offline? what would be a microshift use case for configuring routes?
+
+//OCP module, edit with care; Creating an insecure/http route
+include::modules/nw-creating-a-route.adoc[leveloffset=+1]
+
+//Q: HTTP Strict Transport Security routes use projects; can MicroShift users still use these configs without logging in to a project? https://docs.openshift.com/container-platform/4.15/networking/routes/route-configuration.html#nw-enabling-hsts_route-configuration
+
+//Q: The sticky state stuff didn't seem applicable; lmk if it should be here? https://docs.openshift.com/container-platform/4.15/networking/routes/route-configuration.html#nw-using-cookies-keep-route-statefulness_route-configuration
+
+//OCP module, edit with care
+include::modules/nw-path-based-routes.adoc[leveloffset=+1]
+
+//Q: Can we use this IngressController with MicroShift? Or do we need to edit out references to that and just use the `Route` CR?
```

Comment:
IngressController is a CRD that belongs to OpenShift, MicroShift does not include and it can not be used. We can only mention `Route` from that document.